### PR TITLE
Fix rm to behave properly deleting prefixes

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: cmd/rm-main.go [hunk 1]
```diff
@@ -259,24 +258,23 @@ func removeRecursive(url string, isIncomplete bool, isFake bool, olderThan, newe
...
 
-		// list internally mimics recursive directory listing of object prefixes for s3 similar to FS.
-		// The rmMessage needs to be printed only for actual objects being deleted and not prefixes.
-		if !(content.Time.IsZero() && strings.HasSuffix(urlString, string(filepath.Separator))) {
-			printMsg(rmMessage{
-				Key:  targetAlias + urlString,
-				Size: content.Size,
-			})
-		}
+		printMsg(rmMessage{
```
Comment (Praveenrajmani) [inline cmd/rm-main.go]: Should we print the mssage if `content.Time.IsZero()` ?

Comment (harshavardhana) [reply cmd/rm-main.go]: why should we not?

Comment (Praveenrajmani) [reply cmd/rm-main.go]: :+1

Comment (Praveenrajmani) [reply cmd/rm-main.go]: I thought `strings.HasSuffix` is enough to find if the name is a dir or not

**Pull request merged**